theano/tensor/nnet/tests/test_nnet.py

```diff
@@ -1295,7 +1309,7 @@ def test_argmax_pushdown():
         fgraph = gof.FunctionGraph(
                 [x],
                 [out])
-        assert hasattr(fgraph.outputs[0].tag, 'trace')
+        assert check_stack_trace(fgraph, ops_to_check='all')
```

Comment:
So, that fgraph is not optimized yet at that point, and `local_argmax_pushdown` is not going to be applied here anyway, so you can drop this assert.